corporate/lib/stripe.py

```diff
@@ -209,7 +209,9 @@ class BillingError(JsonableError):
     def __init__(self, description: str, message: Optional[str] = None) -> None:
         self.error_description = description
         if message is None:
-            message = BillingError.CONTACT_SUPPORT.format(email=settings.ZULIP_ADMINISTRATOR)
+            message = getattr(BillingError.CONTACT_SUPPORT, "format")(
```

Comment:
I feel like `getattr` isn't a pattern we want to be using over accessing values directly...

Reply:
Another way will probably be forcing these into `str` and then call methods on it. Since the type checker does not know what attributes will be available on the `Promise`, `getattr` is a way to get around with this.